xcp_d/workflow/anatomical.py

```diff
@@ -1,784 +1,1371 @@
+                ),
+                name="combine_xfms",
+                mem_gb=mem_gb,
+                n_procs=omp_nthreads,
+            )
+            combine_inv_xfms = pe.Node(
+                antsapplytransforms(
+                    reference_image=mnitemplate,
+                    interpolation="LanczosWindowedSinc",
+                    print_out_composite_warp_file=True,
+                    output_image="ants_composite_inv_xfm.nii.gz",
+                ),
+                name="combine_inv_xfms",
+                mem_gb=mem_gb,
+                n_procs=omp_nthreads,
+            )
+            # use C3d to separate the combined warpfield xfm
+            # into x, y, and z components
+
+            get_xyz_components = pe.Node(
+                C3d(
+                    is_4d=True,
+                    multicomp_split=True,
+                    out_files=["e1.nii.gz", "e2.nii.gz", "e3.nii.gz"],
+                ),
+                name="get_xyz_components",
+                mem_gb=mem_gb,
+                n_procs=omp_nthreads,
+            )
+
+            get_inv_xyz_components = pe.Node(
+                C3d(
+                    is_4d=True,
+                    multicomp_split=True,
+                    out_files=["e1inv.nii.gz", "e2inv.nii.gz", "e3inv.nii.gz"],
+                ),
+                name="get_inv_xyz_components",
+                mem_gb=mem_gb,
+                n_procs=omp_nthreads,
+            )
+            # select x-component after separating warpfield above
+            select_x_component = pe.Node(
+                niu.Select(index=[0]),
+                name="select_x_component",
+                mem_gb=mem_gb,
+                n_procs=omp_nthreads,
+            )
+
+            select_inv_x_component = pe.Node(
+                niu.Select(index=[0]),
+                name="select_inv_x_component",
+                mem_gb=mem_gb,
+                n_procs=omp_nthreads,
+            )
+
+            # select y-component
+            select_y_component = pe.Node(
+                niu.Select(index=[1]),
+                name="select_y_component",
+                mem_gb=mem_gb,
+                n_procs=omp_nthreads,
+            )
+
+            select_inv_y_component = pe.Node(
+                niu.Select(index=[1]),
+                name="select_inv_y_component",
+                mem_gb=mem_gb,
+                n_procs=omp_nthreads,
+            )
+
+            # select z-component
+            select_z_component = pe.Node(
+                niu.Select(index=[2]),
+                name="select_z_component",
+                mem_gb=mem_gb,
+                n_procs=omp_nthreads,
+            )
+
+            select_inv_z_component = pe.Node(
+                niu.Select(index=[2]),
+                name="select_inv_z_component",
+                mem_gb=mem_gb,
+                n_procs=omp_nthreads,
+            )
+
+            # reverse y-component of the warpfield

```

Comment:
do you have a reference for this? It sounds right but would be good to have a link to someone confirming it